src/Nox.Types/Types/Image/Image.cs

```diff
@@ -10,7 +10,7 @@ namespace Nox.Types;
 /// <summary>
 /// Represents an image value object that encapsulates image-related information.
 /// </summary>
-public sealed class Image : ValueObject<(string Url, string PrettyName, int SizeInBytes), Image>
+public sealed class Image : ValueObject<(string Url, string PrettyName, ulong SizeInBytes), Image>
```

Comment:
Let's please change the underlying type for SizeInBytes to uint for both Image and File, as it was confirmed with Ricardo.

Reply:
done